resources/lang/id/forum.php

```diff
@@ -0,0 +1,268 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'pinned_topics' => 'Topik yang Disematkan',
+    'slogan' => 'Bermain sendiri itu berbahaya.',
+    'subforums' => 'Subforum',
+    'title' => 'osu!community',
+
+    'covers' => [
+        'create' => [
+            '_' => 'Pasang gambar sampul',
+            'button' => 'Unggah gambar',
+            'info' => 'Ukuran gambar yang optimal adalah :dimensions. Anda juga dapat meletakkan gambar di sini untuk mengunggah.',
+        ],
+
+        'destroy' => [
+            '_' => 'Hapus gambar sampul',
+            'confirm' => 'Apakah Anda yakin akan menghapus gambar sampul ini?',
+        ],
+    ],
+
+    'email' => [
+        'new_reply' => '[osu!] Balasan baru dari topik ":title"',
+    ],
+
+    'forums' => [
+        'topics' => [
+            'empty' => 'Topik tidak ditemukan!',
+        ],
+    ],
+
+    'post' => [
+        'confirm_destroy' => 'Yakin menghapus post?',
+        'confirm_restore' => 'Yakin mengembalikan post?',
+        'edited' => 'Terakhir disunting oleh :user :when, total disunting sebanyak :count kali.',
+        'posted_at' => 'Diposting pada :when',
```

Comment:
Without "pada"? (also in `news.php`).

![](https://s.myconan.net/2018-05/firefox_2018-05-11_18-33-18.png)